legacy/firmware/config.h

```diff
@@ -108,7 +117,10 @@ bool config_getPassphraseProtection(bool *passphrase_protection);
 bool config_getHomescreen(uint8_t *dest, uint16_t dest_size);
 void config_setHomescreen(const uint8_t *data, uint32_t size);
 
-const uint8_t *session_getSessionId(void);
+uint8_t session_findLRU(void);
+uint8_t session_findSession(const uint8_t *sessionId);
+uint8_t session_findEmpty(void);
```

Comment:
Shouldn't these functions be for internal use only? Or is there some bigger picture I am not seeing?

Reply:
Indeed, they should. I have a bad habit of putting everything to config.h automatically. Will move to config.c.